update.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# the below will work assuming no other custom modifications have been made.
+rm -rf node_modules/
+git checkout package.json package-lock.json
+git pull
+npm install && npm update && npm dedupe
```

Comment:
should we care about updating scripts here, too?
should it be mentioned in the README?

Reply:
Since there are a lot of questions about getting recent updates, it would be a nice step to supply an update script. but you are right, there should be a reference from the README.md file to this script.

@shr00mie on my opinion it would be perfect to supply also a bat file to update, since zenbot also tries to support native windows support. you could use rmdir instead of rm (for the bat file), rest should be the same.

Reply:
@Shawn8901 *puts on elitist linux hat while submitting this post on a windows7 box*

Reply:
@DeviaVir it was just a thought while discussing stuff with @LoneWolf345 in regards to ways to minimize people reporting issues. i would concur with placing it in README, but for the fact that it would require noobs to read the document to begin with, which...well. just go in #general.

Reply:
alternative would be to build it in as a command so you could run ./zenbot update...maybe have it do a checkout for all affected files based on git response and then update...but that's more effort/time than i have at the moment. at least if it's in the base directory with +x already set, it might allow users, especially those not on the dev side, to just really quickly be able to do a full module wipe, pull, and install.